website/settings/local-dist.py

```diff
@@ -59,6 +59,7 @@
 MAIL_SERVER = 'localhost:1025'  # For local testing
 MAIL_USERNAME = 'osf-smtp'
 MAIL_PASSWORD = 'CHANGEME'
+MAILGUN_API_KEY = 'mailkimp'
```

Comment:
Is this just for tests? If so, then you should be able to let this default to `None` and override `settings.MAILGUN_API_KEY` in whatever tests you need.